Scripts/Python/nb01RPSGame.py

```diff
@@ -380,6 +381,10 @@ def OnSDLNotify(self, VARname, SDLname, playerID, tag):
         else:
             raise RuntimeError("Got an SDL notify for {}, but no CB".format(VARname))
 
+    def OnControlKeyEvent(self,controlKey,activeFlag):
+        if controlKey in [PlasmaControlKeys.kKeyMoveBackward, PlasmaControlKeys.kKeyRotateLeft, PlasmaControlKeys.kKeyRotateRight, PlasmaControlKeys.kKeyExitMode] and activeFlag:
+            PtYesNoDialog(self.key, "Do you want to leave the Ahyoheek game?")
```

Comment:
This will work for now, but we probably want to move toward a localized entry before merging this.